cmd/cassandra-writer/main.go

```diff
@@ -69,19 +70,21 @@ func main() {
 		log.Fatalf(err.Error())
 	}
 
-	b, err := broker.New(cfg.natsURL)
+	nc, err := nats.Connect(cfg.natsURL)
```

Comment:
The main question is - do we want this really? That for every new broker we put this connecting logic in every adapter? Why not solving once and for all in a shared package?

Reply:
It won't change a lot of even if we do that. If we move connection logic to, for example, `pubsub/nats` method `NewPub(..)`, we'll need to:

- return an error alongside with the implementation
- enrich returned interface with `Close` method, so that we can call defer(ifc.Close())
- check returned error `if err != nil ...`

Reply:
Would it be hard to have a generic full wrapper? I would prefer that then having a partial wrapping for every broker.

Reply:
No, we can do that. It won't help too much, but it will mask the underlying broker specifics (nats.Connect).